src/hotspot/share/gc/shared/gc_globals.hpp

```diff
@@ -131,6 +131,12 @@
   product(bool, UseShenandoahGC, false,                                     \
           "Use the Shenandoah garbage collector")                           \
                                                                             \
+  product(ccstr, ErgonomicsProfile, "shared",                               \
+          "Ergonomics profile to use. "                                     \
+          "Options include \"shared\" for when the JVM is running "         \
+          "in a shared environment and \"dedicated\" for when the "         \
+          "JVM is running on a system with dedicated resources")            \
```

Comment:
I feel this needs some real world example else folks may assume "Hey yeah, of course I'm dedicated"

Reply:
Fixed in b254d08